src/products/components/ProductMedia/ProductMedia.tsx

```diff
@@ -209,42 +219,87 @@ const ProductImages: React.FC<ProductImagesProps> = props => {
     <Card className={classes.card}>
       <CardTitle
         title={intl.formatMessage({
-          defaultMessage: "Images",
+          defaultMessage: "Media",
           description: "section header"
         })}
         toolbar={
           <>
             <Button
-              onClick={() => upload.current.click()}
-              disabled={loading}
+              onClick={() => setPopperOpenStatus(true)}
               variant="text"
               color="primary"
               data-test="button-upload-image"
+              ref={anchor}
             >
-              {intl.formatMessage(commonMessages.uploadImage)}
+              {intl.formatMessage({
+                defaultMessage: "Upload",
+                description: "modal button upload"
+              })}
             </Button>
+
+            <Popper
+              open={popperOpenStatus}
+              anchorEl={anchor.current}
+              transition
+              placement="bottom-end"
+            >
+              {({ TransitionProps }) => (
+                <Grow {...TransitionProps}>
+                  <Paper>
+                    <ClickAwayListener
+                      onClickAway={() => setPopperOpenStatus(false)}
+                      mouseEvent="onClick"
+                    >
+                      <Menu>
+                        <MenuItem
+                          onClick={() => imagesUpload.current.click()}
+                          data-test="uploadImages"
+                          key="upload-images"
+                        >
+                          {intl.formatMessage({
```

Comment:
Please put messages inside of defineMessage so it's less cluttered in the return